website/project/model.py

```diff
@@ -2300,6 +2303,13 @@ def update_node_wiki(self, name, content, auth):
         )
         new_page.save()
 
+        wiki_changed.send(
+            name,
```

Comment:
@sloria, I guess I was following the decorator examples further down the page: 
Sending and Receiving Data Through Signals

Should I refactor?